iii-directory/README.md

```
@@ -221,25 +221,27 @@ tree-shaped picker iterate `list` rows themselves and indent by
 
-Sixteen functions, all under `directory::*`. All registrations are
+Thirteen functions, all under `directory::*`. All registrations are
 namespace-clean; this worker is intentionally agnostic to MCP and any
 other adapter.
 
-### `directory::skills::*` (filesystem reader)
+### `directory::skills::*` (filesystem reader + editor)
 
 | Function ID | Description |
 |---|---|
 | `directory::skills::download` | Pull markdown into `skills_folder`. Either `{repo, skill, branch?}` (defaults `branch=main`) or `{worker, version?|tag?}` (defaults `tag=latest`). |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)